integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -154,28 +153,121 @@ The REST API Gateway ensures all requests are authenticated with SigV4. Any Infl
 
 ## IAM Permissions
 
-### IAM Deployment Permissions
+The following permissions are the least required permissions for deploying and executing the connector. These permissions assume the stack is named `InfluxDBTimestreamConnector`.
 
-[//]: # (TODO: Update deployment policy with least privilege once REST API Gateway deployment has been tested.)
+### IAM Deployment Permissions
 
 The following is the least privileged IAM permissions for deploying the connector.
 
 ```json
 {
     "Version": "2012-10-17",
     "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "apigateway:PATCH"
+            ],
+            "Resource": "arn:aws:apigateway:{region}::/account"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "apigateway:POST",
+                "apigateway:GET"
+            ],
+            "Resource": [
+                "arn:aws:apigateway:{region}::/usageplans",
+                "arn:aws:apigateway:{region}::/usageplans/*"
+            ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "sqs:CreateQueue",
+                "sqs:GetQueueAttributes"
+            ],
+            "Resource": "arn:aws:sqs:{region}:{account-id}:InfluxDBTimestreamConnector-LambdaDeadLetterQueue-*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogGroup",
+                "logs:PutRetentionPolicy"
+            ],
+            "Resource": "arn:aws:logs:{region}:{account-id}:log-group:/aws/apigateway/InfluxDBTimestreamConnector-RestApiGatewayLogGroup*:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:DescribeLogGroups"
+            ],
+            "Resource": "arn:aws:logs:{region}:{account-id}:log-group::log-stream:"
```

Comment:
AFAIK DescribeLogGroups does not apply to log-streams. It's account-wide and can be `*` or, technically, `arn:aws:logs:{region}:{account-id}:log-group:`

Reply:
When deploying the stack, an error was returned indicating `log:DescribeLogGroups` was required on this resource. Adding the permission allowed the stack to deploy.

Reply:
Comment was regarding the Resource, not the Action itself..

Reply:
This is the error I get when changing the resource to `arn:aws:logs:{region}:{account-id}:log-group:`:

```
CREATE_FAILED AsyncRestApiGatewayStage
. . . not authorized to perform:
logs:DescribeLogGroups on resource:
arn:aws:logs:us-west-2:460629772345:log-group::log-stream:
because no identity-based policy allows the logs:DescribeLogGroups action
```

Setting the resource to `'*'` works, but AppSec doesn't like wildcards, they want permissions to be as specific as possible, and could tell me to change it.

Reply:
Fair. Sorry for false positive.